environment.yml

```diff
@@ -18,7 +18,6 @@ dependencies:
  - pytest-runner
  - spyder
  - pyzmq
- - ruamel_yaml
  - wrapt
  - pyyaml
```

Comment:
what about this pyyaml ? :)

Reply:
Not really related to this pr but it seems to be unused so lets remove it